api/base/views.py

```diff
@@ -1,10 +1,60 @@
 from django.http import JsonResponse
+from django.core.urlresolvers import resolve, reverse
 from rest_framework.decorators import api_view
 from rest_framework.response import Response
+from rest_framework import generics
 
 from api.users.serializers import UserSerializer
 from .utils import absolute_reverse
 
+class JSONAPIBaseView(generics.GenericAPIView):
+
+    permissions_exempt = True
```

Comment:
(Note for future code reviewers: this was discussed farther down in an outdated section of code)